src/content/docs/en/reference/modules/astro-actions.mdx

```diff
@@ -575,3 +575,31 @@ type ContactResult = ActionReturnType<typeof actions.contact>;
 Represents the result of an action call:
 * on success, `data` contains the output of the action and `error` is `undefined`.
 * on failure, `error` contains an [`ActionError`](#actionerror) with validation errors or runtime errors, and `data` is `undefined`.
+
+### `ActionInputSchema`
+
+<p>
+
+**Type:** `ZodType`
+<Since v="5.16.0" />
+</p>
+
+A utility type that extracts the input's **zod schema type** from [an action handler](#defineaction). This can be useful if you need to reference an [action's validator](#input-validator) input type in your own type definitions.
+
+Even if the action [accepts `form`](#accept-property), `ActionInputSchema` lets you get the input type as `object` rather than [`FormData`](https://developer.mozilla.org/docs/Web/API/FormData).
+
+If the [`input` validator](#input-validator) is omitted, it returns `never`.
+
+The following example uses `ActionInputSchema` on an action named `contact` to:
+* Retrieve the **zod schema type** for the input of the action.
+* Retrieve the expected **input type** of the action's validator.
```

Comment:
```suggestion
* Retrieve the expected input type of the action's validator.
```

Reply:
I think this bolds are important because they highlight the returned data. The sentence is very specific and the important info may get lose in all the words.